docs/DeveloperGuide.md

```diff
@@ -701,44 +652,153 @@ testers are expected to do more *exploratory* testing.
 
-   1. Test case: `dc 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+Test case: `find Zachary`
 
-   1. Other incorrect delete commands to try: `dc`, `dc x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+Expected Output in the Displayed Person List: The list is empty.
 
-1. _{ more test cases …​ }_
+Expected Output in the Result Display: Message states “0 persons listed”.
 
-### Saving data
+### Mark attendance
 
-1. Dealing with missing/corrupted data files
+Prerequisite: There is at least 1 person in the Displayed Person List.
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+Test case: `ma w=1 abs=1`
 
-1. _{ more test cases …​ }_
+Expected Output in the Result Display: A message that informing that the first person in list has been marked as absent.
+
+### List persons in a class/ View attendance for a class
+
+Prerequisite: There is at least 1 person with the class "T42". They should be marked as absent.
+
+Test case: `ls T42`
+
+Expected Output in the Displayed Person List: All persons from the class "T42" are displayed.
+
+Expected Output in the Result Display: Message states the number of persons displayed.
+
+Expected Output in the Side Panel Display: The list of absentees from the class "T42" are displayed by week.
+
+### Adding a note to a person
+
+Prerequisite: There is at least 1 person in the Displayed Person List. The same person is being viewed in the Side Panel Display.
+
+Test case: `an 1 note=very hardworking!`
+
+Expected Output in the Result Display: Message states the details of the first person in the list, as well as the note added.
+
+Expected Output in the Side Panel Display: The new note is displayed under the notes section of the person.
+
+### Deleting a note from a person
+
+Prerequisite: There is 1 person with 1 note in the Displayed Person List. The same person is being viewed in the Side Panel Display.
+
+Test case: `dn 1 i=1`
+
+Expected Output in the Result Display: Message states the details of the first person in the list, as well as the note deleted.
+
+Expected Output in the Side Panel Display: The first note is removed from the notes section of the person.
+
+### Clearing the application data
+
+Test case: `clear`
+
+Expected Output in the Displayed Person List: Nothing is displayed.
+
+Expected Output in the Side Panel Display: Nothing is displayed.
+
+Expected Output in the Result Display: A message stating that app data has been cleared.
+
+### Help
+
+Test case: `help`
+
+Expected Output: A window shows up and provides the link to the User Guide.
+
+### Exit
+
+Test case: `exit`
+
+Expected Output: TA Toolkit closes.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Planned Enhancements
+## Appendix: Planned Enhancements
+
```

Comment:
I believe the first line below planned enhancements needs to have team size, as we are only allowed to have TEAM_SIZE * 2 number of planned enhancements i.e. 10 in our case - so if possible would be good to number each enhancement as well, thanks!

smth like 

`Team Size: 5`

Reply:
Oh i see. I was wondering why you put that